docs/INSTALLING.md

```diff
@@ -0,0 +1,71 @@
+# How do I install the Code Corps Ember app?
+
+### Requirements
+
+You will need to install the following: 
+
+- [Node.js/npm](https://docs.npmjs.com/getting-started/installing-node) we use the LTS version (v 4.4.7)
+- bower: `npm install -g bower`
+- ember-cli: `npm install -g ember-cli`
+
+Do you use other versions of node.js? Check out [nvm](https://github.com/creationix/nvm)
+
+## Clone this repository
+
+You'll want to [clone this repository](https://help.github.com/articles/cloning-a-repository/) with `git clone https://github.com/code-corps/code-corps-ember.git`. If you plan on contributing, you'll want to fork it too!
+
+
+The directory structure will look like the following:
+
+```shell
+code-corps-ember/    # → Root folder for this project
+├── app/
+├── config/
+├── dist/
+├── ...              # → More standard Ember.js files
+...
+```
+
+## Setup your dependencies
+
+> Note: We bind to ports 4200 for Ember and 49152 for livereloading. Make sure you aren't running anything else on these ports.
+
+Go to the `code-corps-ember/` directory and run:
+
+```shell
+npm install
+bower install
+```
+
+Both commands will install the necessary dependencies and will display output regarding those.
+
+### Troubleshooting
```

Comment:
I've included the only major installation error I've run into, has anyone run into others we should highlight?